price-estimator/docker/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM debian:stretch-slim
+# Needed so that price-estimator works with https node urls.
+RUN apt-get update && apt-get --yes install openssl ca-certificates
+COPY target/release/price-estimator /bin/
+ADD docker/rust/tini /tini
+RUN chmod +x /tini
+ENTRYPOINT ["/tini", "--"]
+CMD price-estimator
```

Should this be:
```suggestion
CMD ["price-estimator"]
```

From the docker reference: https://docs.docker.com/engine/reference/builder/#cmd

> The `CMD` instruction has three forms:
> - `CMD ["executable","param1","param2"]` (exec form, this is the preferred form)
> - `CMD ["param1","param2"]` (as default parameters to ENTRYPOINT)
> - `CMD command param1 param2` (shell form)
>
> [..]
>
> If `CMD` is used to provide default arguments for the `ENTRYPOINT` instruction, both the `CMD` and `ENTRYPOINT` instructions should be specified with the JSON array format.